core/src/test/kotlin/com/alecstrong/sql/psi/core/FixturesTest.kt

```diff
@@ -85,20 +85,15 @@ class FixturesTest(val dialect: DialectPreset, val name: String, val fixtureRoot
 
   companion object {
     private val dialects = mapOf(
-        DialectPreset.SQLITE_3_18 to arrayOf("src/test/fixtures", "src/test/fixtures_upsert_not_supported"),
-        DialectPreset.SQLITE_3_24 to arrayOf("src/test/fixtures", "src/test/fixtures_sqlite_3_24"),
-        DialectPreset.SQLITE_3_25 to arrayOf("src/test/fixtures", "src/test/fixtures_sqlite_3_24", "src/test/fixtures_sqlite_3_25"),
-        DialectPreset.MYSQL to arrayOf("src/test/fixtures_mysql"),
-        DialectPreset.HSQL to arrayOf("src/test/fixtures_hsql"),
-        DialectPreset.POSTGRESQL to arrayOf("src/test/fixtures_postgresql")
+        DialectPreset.SQLITE_3_18 to arrayOf("src/test/fixtures")
     )
 
     @Suppress("unused") // Used by Parameterized JUnit runner reflectively.
     @Parameters(name = "{0}: {1}")
     @JvmStatic fun parameters() = dialects.flatMap { (dialect, fixtureFolders) ->
       fixtureFolders.flatMap { fixtureFolder ->
         File(fixtureFolder).listFiles()
-            .filter { it.isDirectory }
+            .filter { it.isDirectory && it.name == "bind-failures" }
```

Comment:
Pardon my harness, i.e. is there a better way to do this?

Reply:
to run a single test what i usually do is run all tests, and then right click the specific one I want from the intellij run panel and run just the one test. It's not a great flow, this works too